src/Tasks/Microsoft.NET.Build.Tasks/Resources/Strings.resx

```diff
@@ -225,4 +225,16 @@
   <data name="NU1012" xml:space="preserve">
     <value>Dependency conflict. '{0}' expected '{1}' but received '{2}'</value>
   </data>
+  <data name="DuplicateItemsDefaultExplanation" xml:space="preserve">
+    <value> The .NET SDK includes {0} items from your project directory by default.</value>
+  </data>
+  <data name="DuplicateItemsError" xml:space="preserve">
+    <value>Duplicate {0} items were included.{1}  The duplicate items were: {2}</value>
+  </data>
+  <data name="DuplicateItemsHowToFix" xml:space="preserve">
+    <value> You can either remove these items from your project file, or set the '{0}' property to '{1}' if you want to explicitly include them in your project file.{2}</value>
```

Comment:
I suspect that the experiences of https://github.com/dotnet/sdk/issues/623 and https://github.com/dotnet/roslyn-project-system/issues/1126 indicate that we should also promote the `Update` and `Remove` mechanisms here. But that makes a very long error, which isn't ideal.